third_party/gpus/find_rocm_config.py

```diff
@@ -82,7 +82,22 @@ def rocm_version_numbers(path):
     patch = int(version_numbers[2].split("-")[0])
     return major, minor, patch
 
-  major, minor, patch = rocm_version_numbers(rocm_install_path)
+  def rocm_version_numbers_post_rocm45(path):
+    version_file = os.path.join(path, "include/rocm/rocm_version.h")
+    if not os.path.exists(version_file):
+      raise ConfigError('ROCm version file "{}" not found'.format(version_file))
```

Comment:
I would update this message to from "ROCm version file X not found. Trying alternative approach ....".

I am afraid for builds previous to rocm4.5, the warning message would be ambiguous.

Reply:
good point....psuhed out the update